xformers/components/multi_head_dispatch.py

```diff
@@ -210,7 +210,7 @@ def check(t, name):
             v = reshape_fn(v, B, S_K, self.num_heads, self.dim_k)
 
         # Self-attend
-        y = self.attention(q=q, k=k, v=v, **kw_mask_args)
+        y = self.attention(q, k, v, **kw_mask_args)
```

Comment:
I think that works, since all the mechanisms need to inherit from the base class, and the base class uses qkv ([see](https://github.com/facebookresearch/xformers/blob/main/xformers/components/attention/base.py#L71)). If it was not for the downstream issue for you I would have kept the keywords, but I think that it's a good enough reason, xformers does not exist in a vacuum and being interoperable with the AI ecosystem is worth something..